config/environments/development.rb

```diff
@@ -107,3 +107,5 @@ def yarn_integrity_enabled?
     Bullet.rails_logger = true
   end
 end
+
+Rails.application.routes.default_url_options = { host: Rails.application.config.app_domain }
```

Comment:
I added `default_url_options` to be able to use routes in view objects. I used to configure `default_url_options` for my projects previously. Please, notify me if we should configure it differently or have good reasons not to configure.

Reply:
Nice